department-of-veterans-affairs/caseflow
# Only allocate CO dockets once per week

Resolves https://vajira.max.gov/browse/CASEFLOW-252

### Description
Previously, we'd allocate at most 4 docket days for CO hearing per week. This PR changes this to only create at most 1 docket day for CO hearing per week. We and the branch chose Wednesday for the day unless wednesday is a holiday, weekend or not available in which case we pick another non-holiday. 

- write new test
- update old test expectations

### Testing Plan
1. Login as `BVASYELLOW` and visit http://localhost:3000/hearings/schedule/build/upload
2. Upload `blankRoSpreadsheet.xlsx` for `RO/CO hearings`; NOTE: you can find `blankRoSpreadsheet.xlsx` within caseflow dir;
   You could also upload `validRoSpreadsheet.xlxs` but start and end dates would have to be within 1/1/18 - 6/1/18
3. Click `Continue` and ensure that correct number of CO docket days would be created for the days you've provided
